src/main/scala/org/broadinstitute/dsde/firecloud/utils/RestJsonClient.scala

```diff
@@ -112,6 +112,9 @@ trait RestJsonClient extends FireCloudRequestBuilding with PerformanceLogging {
       response.status match {
         case s if s.isSuccess =>
           Unmarshal(response.entity).to[T].recoverWith {
+            case de: DeserializationException =>
+              throw new FireCloudExceptionWithErrorReport(
+                ErrorReport(s"could not deserialize response: ${de.msg}"))
```

Comment:
the most common case of an error in unmarshalling is `DeserializationException`; this bubbles up the underlying error.